static/robots.txt

```diff
@@ -0,0 +1,3 @@
+User-agent: *
+Disallow:
```

Comment:
Don't we want:
```
User-agent: *
Allow: /
```
I mean it is more clear that we allow all in such syntax 🤔

Reply:
Sure. Both rules would work, but your right about clarity. Update pushed :)